src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasksInfo.cs

```diff
@@ -23,6 +24,8 @@ protected ShardedOngoingTasksHandlerProcessorForGetOngoingTasksInfo([NotNull] Sh
     {
     }
 
+    protected override DatabaseTopology GetTopology(DatabaseRecord record) => record.Sharding.Orchestrator?.Topology;
```

Comment:
It should be invalid to get a `null` here, no?

Reply:
Done (added check in the caller)

Reply:
why? when the Orchestrator topology can be null?